src/main/kotlin/vm/VM.kt

```diff
@@ -151,6 +175,18 @@ class VM(
                         ticksLeft = c.ticksLeft
                     } else fail(E_VERBNF, "verb name must be string")
                 }
+                O_FUNCALL -> {
+                    val name = (next().value as VString).v
+                    val argCount = next().intFromV
+                    val args = mutableListOf<Value>()
+                    repeat (argCount) { args.add(0, pop()) }
+                    verb.symbols[name]?.also { variableID ->
```

Comment:
I see that this will allow invoking lambdas stored as vars, but is there a way to invoke lambdas that are stored in properties (`someVObj.somePropThatHoldsVFun(x)`)?  I am not well-versed enough with the parser/VM to understand if that is something you'd get for free.

Reply:
Sadly it does not!  I will explain in meatspace.  My parser fu is not that bigbrained.